java/ql/src/experimental/Security/CWE/CWE-295/JXBrowserWithoutCertValidation.qhelp

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>JXBrowser is a Java library that allows to embed the Chromium browser inside Java applications.
+The version 6.x.x by default ignores any HTTPS certificate errors thereby allowing man-in-the-middle attacks.
+</p>
+</overview>
+
+<recommendation>
+<p>Do either of these:
+<li>Update to version 7.x.x as it now correctly rejects certificate errors by default.</li>
```

Comment:
Can we look at e.g. a pom.xml to check if this is the case?

Reply:
This is not needed, see https://github.com/github/codeql/pull/4945/files#diff-a2bfa99015855306e4ee47d13db685b82326ff514cdba0088d4c2e6686ef5c8eR15-R16

> This query is version specific to JXBrowser 6.x.x. The version is indirectly detected.
 In version 6.x.x the `Browser` class is in a different package compared to version 7.x.x.

Reply:
Aha, nice